calcom/cal.com
# Display correct time format on availability page

## What does this PR do?



This PR displays the proper time format on the availability page.

Fixes #2082



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Go to create a booking and choose a date
- The times displayed should be in 12h or 24h depending on your location
- In `timeFormat.ts` and comment out line 11.
- You can now change `locale` to "en-US" for 12h time or "en-GB" for 24h time

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
